Library/Formula/ddclient.rb

```diff
@@ -3,8 +3,8 @@
 class Ddclient < Formula
   desc "Update dynamic DNS entries"
   homepage 'http://sourceforge.net/p/ddclient/wiki/Home'
-  url 'https://downloads.sourceforge.net/project/ddclient/ddclient/ddclient-3.8.2/ddclient-3.8.2.tar.bz2'
-  sha1 '8c887c06a580773c48a1eaced82a08c3077e1325'
+  url 'https://downloads.sourceforge.net/project/ddclient/ddclient/ddclient-3.8.3/ddclient-3.8.3.tar.bz2'
+  sha256 '5f2546687faf82a44c1addf058ef23bd29efba163c7fc1e9660f236aa092e9e51d301320cfdb72b78f2e65860386c30d85773d791f7c298d4ce42ba7dcdb4d2d'
```

Comment:
This is too long: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/29369/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_ddclient/